src/components/content-region/index.jsx

```diff
@@ -15,7 +15,7 @@ const COMPONENTS = {
 			const url = new URL(props.href, location.origin);
 
 			const prefetchAndPreload = () => {
-				if (props.href.startsWith('/repl?code')) {
+				if (props.href.startsWith('/repl')) {
```

Comment:
Not quite sure why I had originally appended the query param like that -- I'm not sure if we directly link to the REPL anywhere in our guide, so it probably doesn't matter, but it's wholly unnecessary. The REPL can be preloaded w/ or w/out a link to a particular code example.